src/Castle.Core/Core/Internal/InternalsVisible.cs

```diff
@@ -21,21 +21,27 @@ public class InternalsVisible
 		///   <c>[assembly: InternalsVisibleTo(CoreInternalsVisible.ToCastleCore)]</c>
 		/// </summary>
 		public const string ToCastleCore =
-#if FEATURE_STRONGNAME
 			"Castle.Core, PublicKey=002400000480000094000000060200000024000052534131000400000100010077F5E87030DADCCCE6902C6ADAB7A987BD69CB5819991531F560785EACFC89B6FCDDF6BB2A00743A7194E454C0273447FC6EEC36474BA8E5A3823147D214298E4F9A631B1AFEE1A51FFEAE4672D498F14B000E3D321453CDD8AC064DE7E1CF4D222B7E81F54D4FD46725370D702A05B48738CC29D09228F1AA722AE1A9CA02FB";
-#else
-			"Castle.Core";
-#endif
 
 		/// <summary>
 		///   Constant to use when making assembly internals visible to proxy types generated by DynamicProxy. Required when proxying internal types.
 		///   <c>[assembly: InternalsVisibleTo(CoreInternalsVisible.ToDynamicProxyGenAssembly2)]</c>
 		/// </summary>
 		public const string ToDynamicProxyGenAssembly2 =
-#if FEATURE_STRONGNAME
 			"DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7";
-#else
-			"DynamicProxyGenAssembly2";
-#endif
+

```

Comment:
Another way is to programmatically read the public key from the embedded DynProxy.snk.  However since the key remain relatively stable, and if it changes, these constants need to be updated too.  So I don't feel too much value in reading public key from the embedded snk file.

Any suggestion for the proper place for this field?  It is not too bad to have these public keys to stay together though.